cmake/modules/coverage.cmake

```diff
@@ -152,7 +152,7 @@ function(set_target_for_coverage_local target_name)
   add_custom_target(${name}
 
     # Cleanup lcov
-    COMMAND ${LCOV_EXECUTABLE} --directory . --zerocounters
+    # COMMAND ${LCOV_EXECUTABLE} --directory . --zerocounters
```

Comment:
Since this file is used to generate coverage for other parts for the project as well, do you think not resetting the counters can have a negative effect? Do we need to have a separate version of this just for ASE?

Reply:
@nakulkorde I think it should still work. @abelardojarab , can you comment on this?

Reply:
mm i think best should be add a flag whose default behavior (if flag is not specified) is not to flush coverage counters.

Reply:
sounds good